docs/internal/runtime-upgrade.md

```diff
@@ -0,0 +1,208 @@
+# Runtime upgrade with the Runtime Provisioner
+
+## Upgrade Runtime
+
+Runtime Provisioner allows to upgrade Kyma Runtime using the GraphQL API. To upgrade a Runtime to version 1.11.0 with the standard Runtime components use the following mutation:
```

Comment:
```suggestion
Runtime Provisioner allows you to upgrade Kyma Runtime using the GraphQL API. To upgrade a Runtime to the 1.11.0 version with the standard Runtime components, use the following mutation:
```

Reply:
I'll add the comma, but I'll stick to the original version description as the one in the suggestion AFAIK is not really how it's written in the IT context.